cmd/open.go

```diff
@@ -0,0 +1,62 @@
+package cmd
+
+import (
+	"strings"
+
+	"github.com/CircleCI-Public/circleci-cli/git"
+	"github.com/jtacoma/uritemplates"
+	"github.com/pkg/browser"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+func projectUrl(remote *git.Remote) (string, error) {
+
+	template, err := uritemplates.Parse("https://app.circleci.com/{vcs}/{org}/{project}/pipelines")
```

Comment:
Is the new dependancy, values map and extra error handling really better than..

```
func projectUrl(remote *git.Remote) string {
	return fmt.Sprintf("https://app.circleci.com/%s/%s/%s/pipelines",
		url.PathEscape(strings.ToLower(string(remote.VcsType))),
		url.PathEscape(remote.Organization),
		url.PathEscape(remote.Project))
}
```

Reply:
nope, I added it at @glenjamin 's suggestion. I've removed it.